### ISSUE TITLE
Inline radiobuttons

### ISSUE THREAD

**Issue by lobosan** [djhi/meteor-autoform-materialize#23]
Inline radio buttons errors with label and not rendering inline

**Comment by djhi** [comment]
Thanks for reporting. Can you please add more details and/or a reproduction repo ?

**Comment by lobosan** [comment]
For instance I'm using simple schema, also having trouble with selects

```
Organizaciones = new Mongo.Collection('organizaciones');

Organizaciones.attachSchema(new SimpleSchema({
    diaPreferidoReuniones: {
        type: String,
        label: 'Día preferido para reuniones/talleres',
        autoform: {
            type: 'select-radio-inline',
            defaultValue: 'Ninguno',
            options: function () {
                return [
                    {label: 'Lunes', value: 'Lunes'},
                    {label: 'Martes', value: 'Martes'},
                    {label: 'Miércoles', value: 'Miércoles'},
                    {label: 'Jueves', value: 'Jueves'},
                    {label: 'Viernes', value: 'Viernes'},
                    {label: 'Sábado', value: 'Sábado'},
                    {label: 'Domingo', value: 'Domingo'},
                    {label: 'Ninguno', value: 'Ninguno'}
                ];
            }
        }
    }
}));
```

**Comment by djhi** [comment]
Materialize does not support inline-radios or checkboxes by default. I added support for it in this package but be aware that it might not work with future versions

Fixed in 0.0.20 which will be released today
